### ISSUE TITLE
Hydra logo licensing

### ISSUE THREAD

**Issue by tpluscode** [HydraCG/Specifications#181]
@lanthaler is there an explicit license for the hydra logo?

I'd expect it to be CC0 or similar permissive license but I don't see it mentioned anywhere.

**Comment by lanthaler** [comment]
I own the rights to the logo. It's not licensed under CC0 or any other permissive license so you'll need to ask me if you want to use it for anything but the homepage

**Comment by tpluscode** [comment]
Well, that's unfortunate. Is there a specific reason you have it that way?

And more importantly, are you intent on keeping exclusive rights to it?

**Comment by lanthaler** [comment]
My name is strongly associated with that brand and I'm not comfortable to give control over it up just yet. So for now I intent to keep the rights to see where all of this is going. We can revisit at a later point.

**Comment by alien-mcl** [comment]
I agree with @asbjornu - I know that in some cases logo and hydra name can be abused, but having to ask for permission every time somebody would like to use it on their i.e. presentation or piece of software using hydra is no go - people will just won't use it at all.
Alternative would be to have ... another logo under a permissive licence, but this will ruin the hydra brand, thus I wouldn't like to use this approach.

**Comment by lanthaler** [comment]
> Could you please consider offering the logo under a permissive license?

Sure... but I'd like to discuss a concrete proposal.

**Comment by asbjornu** [comment]
My preference would be for the logo to be licensed under [CC-BY](https://creativecommons.org/licenses/by/4.0/), so it can be adapted and transformed into other logos for different implementations of the Hydra protocol.

I'd also like the licensing to **allow commercial use**, so companies that choose to implement Hydra can advertise the fact within their products without causing legal questions related to licensing which are usually resolved by simply not using the logo (instead of asking for permission).

**Comment by lanthaler** [comment]
Yeah, those are exactly the two reasons I initially said no to CC0 or similar permissive licenses. I'm totally fine if people "would like to use it on their i.e. presentation or piece of software using hydra" as suggested by @alien-mcl. None of that should require a special license or asking for permission as it would fall under fair use anyway. I'm not fine if people mess with the logo and use it to sell stuff that may has very little to do with Hydra though.

**Comment by tpluscode** [comment]
I believe CC BY-ND fits here

**Comment by asbjornu** [comment]
> I'm not fine if people mess with the logo and use it to sell stuff that may has very little to do with Hydra though.

Thanks for voicing your concerns, @lanthaler. I understand your reasoning around usage that have little or nothing to do with Hydra. To avoid that, we could require affiliation with Hydra of some sort, such as membership of the CG. Would that help, you think?

Regarding the first part of the sentence, say someone created a FOSS implementation of Heracles in Python and wanted to use the Hydra logo in a derived form for it there. Would you oppose this?

**Comment by lanthaler** [comment]
> Thanks for voicing your concerns, @lanthaler. I understand your reasoning around usage that have little or nothing to do with Hydra. To avoid that, we could require affiliation with Hydra of some sort, such as membership of the CG. Would that help, you think?

No, I don't think so as the CG is open to everyone and membership doesn't mean much by itself.

> Regarding the first part of the sentence, say someone created a FOSS implementation of Heracles in Python and wanted to use the Hydra logo in a derived form for it there. Would you oppose this?

Almost certainly not but I'd like to reserve the right to do so. I'd prefer if the Hydra logo would be reserved for Hydra itself and would like to avoid diluting the brand or implying official endorsement by the use thereof (or a slight derivation thereof). Under fair use that tool could obviously advertise its Hydra-compliance by using the official logo (and that shouldn't require to change it in any way).

I hope this clarifies things. Let's discuss it again when the first case that doesn't fall under fair use comes up. I think there are more important discussions to be had right now to move Hydra forward ;-P

**Comment by asbjornu** [comment]
> I hope this clarifies things.

It does, thanks.

> Let's discuss it again when the first case that doesn't fall under fair use comes up.

The problem with keeping status quo is that "fair use" doesn't really work in practice anymore. While the copyright law of US and most other countries permits "fair use", things like DMCA has made it very hard and/or expensive to enforce the right to fair use practice. This causes fear and uncertainty, which actively drives people away from using copyrighted material that isn't clearly licensed for reuse.

I believe [CC-BY-NC-ND](https://creativecommons.org/licenses/by-nc-nd/4.0/) would be better for Hydra than the unclarity of "fair use". Would you oppose that too, @lanthaler?

**Comment by tpluscode** [comment]
I'm not convinced by restricting the use to non-commercial only. Does it mean that a company which publishes a Hydra API would not be allowed to advertise that fact using the official logo?

**Comment by asbjornu** [comment]
@tpluscode, my understanding is that [Creative Commons](https://creativecommons.org/faq/#does-my-use-violate-the-noncommercial-clause-of-the-licenses) leaves that up to interpretation. So we can specify that such use is OK. What wouldn't be OK is for someone to use the Hydra logo as the logo of their commercial product in any way or form; both NC and ND prohibits that.

**Comment by alien-mcl** [comment]
I believe that using either hydra or hydra logo as some other work (i.e. I wrote a piece of software named XYZ and I'll put a hydra logo side by side) would be considered as misuse of the hydra brand - I'd feel bad with it.
But having same XYZ creation and hydra logo put wyth `hydra powered` label is fine - the XYZ is a separate tool that uses hydra and this is made clear.
Unfortunately, I'm not that good with licensing. Still, I'm fine with commercial use when the vocabulary is used. Not sure about the tools created by HydraCG

**Comment by alien-mcl** [comment]
It seems that the issue is quite hot, let me reopen it.

**Comment by lanthaler** [comment]
> I'm not convinced by restricting the use to non-commercial only. Does it mean that a company which publishes a Hydra API would not be allowed to advertise that fact using the official logo?

Depends on how they do it as Karol already outlined. They can't sell it as "Hydra" but they can obviously advertise the fact that they support Hydra. 

> I believe CC-BY-NC-ND would be better for Hydra than the unclarity of "fair use". Would you oppose that too, lanthaler?

No, that would be fine with me as it addresses the two concerns I raised earlier. There doesn't seem to be consensus in the group though. So, let me know if CC-BY-NC-ND works for you guys and I'll release it under that license.

**Comment by tpluscode** [comment]
Fine by me

**Comment by alien-mcl** [comment]
Can we resolve this issue somehow?

**Comment by asbjornu** [comment]
Proposal for resolution: License the logo under CC-BY-NC-ND. If that license proves difficult to make work in practice, we can open a new issue and discuss alternatives. At least the licensing will be explicit and unambiguous, which is an improvement over the current situation, imho.

**Comment by lanthaler** [comment]
Let me know once the CG decided that licensing the logo under CC-BY-NC-ND would resolve this and I'll make the change. I don't want to go back and forth and this and discuss licenses again in a few months.

**Comment by asbjornu** [comment]
@alien-mcl and @tpluscode, can you please write an e-mail to the CG asking for consensus on licensing the logo under CC-BY-NC-ND, referencing this discussion? It would be great to be able to resolve this issue, finally. :)

**Comment by asbjornu** [comment]
[I've hereby written to the CG](https://lists.w3.org/Archives/Public/public-hydra/2025Aug/0000.html). Let's give the discussion there a couple of weeks and conclude this. :)

**Comment by filip26** [comment]
@lanthaler  This kind of approach is exactly why Hydra risks never breaking out of its niche. When energy is wasted on ego-driven, self-centric arguments over trivial details, it doesn’t look like an open standard, it looks like a closed show.

Insisting on control over things that don’t actually move the spec forward is not stewardship, it’s vanity. And that vanity sends the worst possible message to potential adopters: that Hydra is more about protecting egos than solving problems.

Standards only thrive when they lower barriers, encourage broad use, and make outsiders feel welcome. Every time Hydra leans into self-importance over practicality, it loses another chance to be taken seriously.

If the goal is adoption, then ego needs to take a back seat. Otherwise Hydra will remain what it increasingly looks like from the outside: a small, closed show arguing with itself.

**Comment by kephas** [comment]
[CC BY-ND](https://creativecommons.org/licenses/by-nd/4.0/deed.en) would seem fine for the logo, but [CC BY-NC-ND](https://creativecommons.org/licenses/by-nc-nd/4.0/) would mean that no commercial product can advertise its use or conformance with Hydra, something that's explicitly acknowledged in the 
[W3C® Trademark and Servicemark License](https://www.w3.org/Consortium/Legal/trademark-license-19990516.html).

There are precedents but they are pretty narrow in scope: the Semantic Web and SVG "technology buttons" need permission for for-profit use, but in both cases, the logos alone don't (and the [SVG logo](https://www.w3.org/2009/08/svg-logos.html) alone is licensed under [CC BY-SA](https://creativecommons.org/licenses/by-sa/4.0/)). Some logos are licensed under [CC BY](https://creativecommons.org/licenses/by/4.0/) ([HTML5](https://www.w3.org/html/logo/), [WebGPU](https://www.w3.org/2023/02/webgpu-logos.html)).
